docs/pages/database-access/guides/rds.mdx

```diff
@@ -43,74 +55,296 @@ This guide will help you to:
+Database Service. The chart creates a secret containing the join token you
+generated earlier in this guide and mounts it at the path
+`/etc/teleport-secrets/auth-token` on the Database Service pod. The
+configuration above instructs the Database Service to search for its token
+there.
 
-Teleport needs AWS IAM permissions to be able to:
+</TabItem>
+</Tabs>
+
+## Step 3/6. Create IAM policies for Teleport
+
+The Teleport Database Service needs AWS IAM permissions to be able to:
 
 - Discover and register RDS instances and Aurora clusters.
 - Configure IAM authentication for them.
 
-Before you can generate IAM permissions, you must provide the Teleport Database
-Service access to AWS credentials.
+In this step, we will show you how to provide the Teleport Database Service
+access to AWS credentials by using the Teleport configuration file we created
+earlier to automatically create and attach IAM policies:
+
+<Tabs>
+<TabItem label="Linux Host">
+
+Follow these instructions on your Linux host.
 
 (!docs/pages/includes/aws-credentials.mdx service="the Database Service"!)
 
 (!docs/pages/includes/database-access/aws-bootstrap.mdx!)
 
+</TabItem>
+<TabItem label="Kubernetes Cluster">
+
+Follow these instructions on your local workstation, where you will generate IAM
+policies for the Teleport Database Service using the `teleport` binary.
+
+### Obtain AWS credentials
+
+You will need to authenticate the `teleport` binary to your AWS account by
+providing credentials to its AWS client.
+
+Obtain values for the following environment variables from your organization and
+export these variables within your shell:
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_DEFAULT_REGION`
+
+<Details title="Have multiple sources of AWS credentials?">
+
+Teleport's AWS client loads credentials from different sources in the following
+order:
+
+- Environment Variables
+- Shared credentials file
+- Shared configuration file (Teleport always enables shared configuration)
+- EC2 Instance Metadata (credentials only)
+
+While you can provide AWS credentials via a shared credentials file or shared
+configuration file, you will need to run the `teleport` binary with the
+`AWS_PROFILE` environment variable assigned to the name of your profile of
+choice. 
+
+If you have a specific use case that the instructions above do not account for,
+consult the documentation for the [AWS SDK for
+Go](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/) for a detailed
+description of credential loading behavior.
+
+</Details>
+
+### Generate and attach IAM policies
+
+Create IAM policies for the Teleport Database Service so you can attach them to
+its IAM identity. 
+
+For the purpose of this guide, retrieve the IAM role associated with your EKS
+node group, which we will attach our newly generated policies to.
+
+<Admonition title="Do not do this in production" type="warning">
+
+While we are attaching policies to the EKS node group for simplicity, this
```

Comment:
We should be using a service account, not attaching to a node role.  See https://github.com/gravitational/teleport/discussions/24116

Reply:
@stevenGravy Is it safe to assume that enough readers will have the OIDC provider installed in their cluster to make this a prerequisite? Otherwise, the extra steps are too ungainly to include in this guide, I think.

Reply:
Yes we don't need to go into details on the oidc provider config

Reply:
@stevenGravy I've changed the guide to describe using the OIDC provider, though when I followed these instructions, the Teleport Database Service pod showed this error in its logs: 

```
2023-07-13T19:45:46Z ERRO [IAM]       Failed to auto-configure IAM for Database(Name=example, Type=rds, Labels=map[env:dev teleport.dev/origin:config-file]). error:[
ERROR REPORT:
Original Error: *awserr.baseError WebIdentityErr: failed to retrieve credentials
caused by: AccessDenied: Not authorized to perform sts:AssumeRoleWithWebIdentity
        status code: 403, request id: 6d38d1d1-4f06-4198-afb5-c84502078226
Stack Trace:
        github.com/gravitational/teleport/lib/cloud/aws/identity.go:112 github.com/gravitational/teleport/lib/cloud/aws.GetIdentityWithClient
        github.com/gravitational/teleport/lib/srv/db/cloud/iam.go:218 github.com/gravitational/teleport/lib/srv/db/cloud.(*IAM).getAWSIdentity
        github.com/gravitational/teleport/lib/srv/db/cloud/iam.go:182 github.com/gravitational/teleport/lib/srv/db/cloud.(*IAM).getAWSConfigurator
        github.com/gravitational/teleport/lib/srv/db/cloud/iam.go:249 github.com/gravitational/teleport/lib/srv/db/cloud.(*IAM).processTask
        github.com/gravitational/teleport/lib/srv/db/cloud/iam.go:125 github.com/gravitational/teleport/lib/srv/db/cloud.(*IAM).Start.func1
        runtime/asm_amd64.s:1598 runtime.goexit
User Message: WebIdentityErr: failed to retrieve credentials
caused by: AccessDenied: Not authorized to perform sts:AssumeRoleWithWebIdentity
        status code: 403, request id: 6d38d1d1-4f06-4198-afb5-c84502078226] cloud/iam.go:127
 ```
 
 I checked the Service Account annotation and trust policy, and everything seems to be in order. Is there anything I'm missing? Thanks!